cognee/api/v1/add/routers/get_add_router.py

```
@@ -36,17 +36,15 @@
-                                    "data://.data/",
-                                    f"{data.split('/')[-1]}",
-                                )
+                    response = requests.get(data)
+                    response.raise_for_status()
+
+                    if response.status_code == 200:
+                        file_data = await response.content()
+
+                        return await cognee_add(file_data)
+
+                    return JSONResponse(status_code=409, content={"error": str(error)})
```

Ruff (0.8.2)

Undefined name `error`

(F821)